projects/ad9656_fmc/src/app/ad9553.c

```diff
@@ -0,0 +1,244 @@
+/***************************************************************************//**
+ * @file ad9553.c
+ * @brief Implementation of AD9553 Driver.
+ * @author DHotolea (dan.hotoleanu@analog.com)
```

Comment:
It seems that you also added some new drivers with this PR. Separate the drivers in different commits and then just add a commit with the project.

Reply:
I discussed with Dragos Bogdan and we will leave it like it is until there will be another project that will require one of the two drivers. In that situation the setup functions will be modified to be more generic and the drivers will be included in the drivers folder.

Reply:
Ack.